loopin.go

```diff
@@ -222,6 +256,24 @@ func validateLoopInContract(lnd *lndclient.LndServices,
 	return nil
 }
 
+// sendUpdate reports an update to the swap state.
+func (s *loopInSwap) sendUpdate(ctx context.Context) error {
+	info := s.swapInfo()
+	log.Infof("Loop in swap state: %v", info.State)
```

Comment:
While you're here, can we make this s.log? Right now in the logs it's not clear which loop the state belongs to if you have a few on the go.